otp.bash

```diff
@@ -258,6 +258,7 @@ cmd_otp_append() {
 
   local prompt uri
   local path="${1%/}"
+  local prompt="$path"
```

Comment:
You can remove the duplicate `prompt` declaration above.

Reply:
Whoops - good catch, I totally meant to do that.